PythonPonies/bangazon-storefront
# Customer Model w/ test

# Description
This pull request includes the customer model with a test for successful construction of table/customer values. 

## Number of Fixes
N/A

## Related Ticket(s)
N/A

## Problem to Solve
The Customer model is required for several other tests to pass, such as the test for the Products model

## Proposed Changes
Bangazon Storefront will now have the ability to save customer information 

## Expected Behavior
When the model is migrated, the database will include a customer model with the specified fields, and that table will have a user ID. 

## Steps to Test Solution

(In the Terminal)
1. python manage.py makemigrations
2. python manage.py migrate
3. python manage.py test bangazon_storefront 
4. Ensure the test has passed ("OK")
4. In sqlite, check that the Customer table was created 

## Testing

[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

## Documentation
[x ] There is new documentation in this pull request that must be reviewed..
[x ] I added documentation for any new classes/methods

## Deploy Notes
N/A
